test/emit-declaration/emit-declaration.test.js

```diff
@@ -11,3 +11,26 @@ test('Can emit valid declaration', async () => {
 
   expect(declaration).toMatchSnapshot();
 });
+
+test('Can emit valid declaration with sourceMap', async () => {
+  await compiler(require.resolve('./index.js'), { sourceMap: true });
+
+  const declaration = fs.readFileSync(
+    require.resolve('./index.css.d.ts'),
+    'utf-8'
+  );
+
+  expect(declaration).toMatchInlineSnapshot(`
```

Comment:
Used an inline snapshot here because I wanted to have full control over what it was matching against. It also allowed me to add the test first, verify it failed, and then add my fix and re-run the test to verify it passes now. I could not have done that as easily if I generated the snapshots with the broken code.